src/services/email.service.js

```diff
@@ -0,0 +1,53 @@
+const nodemailer = require('nodemailer');
+
+const mailPort = process.env.MAIL_PORT;
+const host = process.env.MAIL_HOST;
+const hostEmail = process.env.HOST_EMAIL;
+const hostID = process.env.HOST_ID;
+const password = process.env.HOST_PASSWORD;
+
+const transporter = nodemailer.createTransport({
+    service: "naver",
+    host: host,
+    port: mailPort,
+    auth: {
+        user: hostID,
+        pass: password
+    }
+})
+
+require('dotenv').config()
+
+const sendResetLink = ({ resetToken, email }) => {
+    let link = `http://localhost:3000/api/users/resetPassword/emailVerify/${resetToken}`;
```

Comment:
## Needs to be changed
This is here just for now as a place holder. However, when the frontend development is finished, change this to the actual URI.